cic/clsp/filters/rekey_and_payment.clsp

```diff
@@ -61,8 +61,11 @@
                         (not (> (ALLOWED_REKEY_TIMELOCK CURRIED_ARGS) (f (r (r curried_args_to_coin)))))
                       )
                   )
-                  ; if we found an even coin then we can exit early as above layer asserts only one even
-                  (r conditions)
+                  (check_even_coin
```

Comment:
Why undo this?

Reply:
In a comment above, I explained how I don't like relying on the layer above for the security of this section of the code.